src/components/dex/Trade.vue

```diff
@@ -328,6 +364,33 @@ export default {
       this.side = side;
       this.$store.commit('setOrderQuantity', '');
     },
+
+    validateQuantity() {
```

Comment:
It would be nice if this were in at least a file that was the same across Mobile and Desktop.